Sources/Yams/Emitter.swift

```diff
@@ -7,24 +7,25 @@
 //
 
 #if SWIFT_PACKAGE
-    import CYaml
+import CYaml
 #endif
 import Foundation
 
-/// Produce YAML String from objects
+/// Produce a YAML string from objects.
 ///
-/// - Parameters:
-///   - objects: Sequence of Object
-///   - canonical: output should be the "canonical" format as in the YAML specification.
-///   - indent: the intendation increment.
-///   - width: the preferred line width. @c -1 means unlimited.
-///   - allowUnicode: unescaped non-ASCII characters are allowed if true.
-///   - lineBreak: preferred line break.
-///   - explicitStart: explicit document start `---`
-///   - explicitEnd: explicit document end `...`
-///   - version: YAML version directive
-/// - Returns: YAML String
-/// - Throws: `YamlError`
+/// - parameter objects:       Sequence of Objects.
+/// - parameter canonical:     Output should be the "canonical" format as in the YAML specification.
+/// - parameter indent:        The intendation increment.
+/// - parameter width:         The preferred line width. @c -1 means unlimited.
+/// - parameter allowUnicode:  Unescaped non-ASCII characters are allowed if true.
+/// - parameter lineBreak:     Preferred line break.
+/// - parameter explicitStart: Explicit document start `---`.
+/// - parameter explicitEnd:   Explicit document end `...`.
+/// - parameter version:       YAML version directive.
+///
```

Comment:
`sortKeys` is missing. Sorry, I forgot to add the `sortKeys` parameter to doc comments on `dump()` and `serialize()` methods.
I'll mark them.